doc/reference/reference_sql/sql_statements_and_clauses.rst

```diff
@@ -620,140 +620,95 @@ causes no error message, although (s1, s1) is probably a user error.
 Table Constraint Definition for foreign keys
 ++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
 
-FOREIGN KEY constraints look like this: |br|
-:samp:`FOREIGN KEY ({referencing-column-name} [, {referencing-column-name}...]) REFERENCES {referenced-table-name} [({referenced-column-name} [, {referenced-column-name}...]]) [MATCH FULL] [update-or-delete-rules]`
+A foreign key is a constraint that can be used to enforce data integrity across related tables.
+A foreign key constraint is defined on the child table that references the parent table's column values.
 
-There is a shorthand: specifying REFERENCES in a :ref:`column definition <sql_column_def_constraint>`.
+Foreign key constraints look like this:
```

Comment:
I see that the described FK constraint is known as the FK tuple constraint. What about the FK field constraint? I'm not sure if the FK description should be duplicated due to different types of constraints, but they have different error descriptions and are described differently in '_space', so at least add a quick overview of that and links to 'tuple constraint' and ' field constraint ' looks like a good idea.

Reply:
Looks like this requires some investigation, created a separate ticket: https://github.com/tarantool/doc/issues/3703